recipes/zeromq/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.definitions["WITH_LIBSODIUM"] = self.options["zmq"].encryption == "libsodium"
```

Comment:
If the package name is `zeromq`... why `zmq` here?

Reply:
Oversight of my part.

Reply:
`WITH_LIBSODIUM` is only used by the test_package project.
It's just a definition to test libsodium integration.